make check-unit generates warnings

```
make check-unit
Makefile:24: warning: overriding recipe for target 'check-unit'
Makefile:16: warning: ignoring old recipe for target 'check-unit'
```

--- a/microcloud/Makefile
+++ b/microcloud/Makefile
@@ -19,10 +19,6 @@ check-unit:
 check-system:
 	cd test && ./main.sh
 
-.PHONY: check-unit
-check-unit:
-	go test ./... -tags test
-
 .PHONY: check-static
 check-static:
 ifeq ($(shell command -v golangci-lint 2> /dev/null),)

